copier.yml

```diff
@@ -113,4 +113,4 @@ _skip_if_exists:
 
 # Tasks to execute after generating or updating a project.
 _tasks:
-    - "git init --initial-branch=main"
+    - "git init . && { git branch -m master main 2>/dev/null || true; }"
```

Comment:
this should be removed from this PR
Even so it does not work on git 1.8 which is the (ancient) version available on epyc.
I do have a solution for that although I think it rather ugly.